src/ex3/index.js

```diff
@@ -0,0 +1,112 @@
+import { Command } from 'commander';
+import Inquirer from 'inquirer';
+import * as ItemManager from './ItemManager.js';
+
+const program = new Command();
+
+function addTasksCli(args,options) {
```

Comment:
make sure you have Prettier configured in your IDE (it's a vs-code extension), and make sure it's configured to format files upon save

https://user-images.githubusercontent.com/9549288/173351346-5b94d353-545d-4634-8ba9-2df1db8bf922.mov

it will make your life easier
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/9549288/173351446-6ea3d50e-a429-48e7-b934-cee9b3b55c55.png">

Reply:
Thanks! i have it, i guess I missed this one